src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -203,7 +197,7 @@ function WorkspaceWorkflowsPage({policy, betas, route, session}: WorkspaceWorkfl
                         />
                         {hasVBA && policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES && (
                             <OfflineWithFeedback
-                                pendingAction={policy?.pendingFields?.reimburserEmail}
+                                pendingAction={policy?.pendingFields?.reimburser}
                                 errors={ErrorUtils.getLatestErrorField(policy ?? {}, CONST.POLICY.COLLECTION_KEYS.REIMBURSER_EMAIL)}
```

Comment:
We need to update the errors too, since it now comes from `policy.errorFields.reimburser`, no?

Reply:
Yes, I missed this one, thanks